_dashboards/discover/multi-data-sources.md

```diff
@@ -132,6 +132,24 @@ The following image shows a date histogram with an interval dropdown list.
 
 <img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-11.jpg" alt="Histogram user interface">
 
+## Selecting multiple data sources in the Dev Tools console
+
+Selecting multiple data sources in the Dev Tools console allows you to work with a broader range of data and gain deeper insights into your code and applications. You can follow these steps:
+
+1. To access the Dev Tools console, open OpenSearch Dashboards and select **Dev Tools** in the main menu.
+2. In the console, input the following query in the editor pane and then select the play button.
+
+    ```json
+    GET /_cat/indices
+    ```
+
+3. From the **DataSource** dropdown menu, select a data source and then query the source.
```

Comment:
Note that the datasource dropdown will appear only when datasource feature is enabled in setting yml file. Like this section `# Modifying the multiple data sources settings` mentions.

Reply:
Added steps to modify the YAML configuration.